pom.xml

```diff
@@ -231,6 +231,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.13.0</version>
+                <configuration>
```

Comment:
why this overriding of jdk versions ? it is already at 17...
I've just checked your PR with the deletion of this two overinf lines and UT et IT are OK.